### DISCUSSION TITLE
[Enhancement] [radio-button] radio-button inside radio-group enhancement

### DISCUSSION THREAD

**Discussion by CeremcoBarbe** [element-plus/element-plus#15976] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `radio-button`

## Description

I would like to enhance the functionality of el-radio-button inside a el-radio-group to make use of the label/value functionality instead of only the label.

Right now when you use a radio button inside a radio group. The label tag on the radio button is used in both the visualization and as a value that gets set on the v-model of the radio group. 
I would like to split this up into a label for visualization and a 'value' that can be used to set the v-model.

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
#15525
